--- a/sn2md/importer.py	2025-11-05 17:34:07
+++ b/sn2md/importer.py	2025-11-05 17:35:45
@@ -61,14 +61,16 @@
 
 
 def create_basic_context(file_basename: str, file_name: str) -> dict:
+    created_at = datetime.fromtimestamp(os.path.getctime(file_name))
     return {
         "file_basename": file_basename,
         "file_name": file_name,
-        "ctime": datetime.fromtimestamp(os.path.getctime(file_name)),
+        "ctime": created_at,
         "mtime": datetime.fromtimestamp(os.path.getmtime(file_name)),
-        "year_month_day": datetime.fromtimestamp(os.path.getctime(file_name)).strftime(
-            "%Y-%m-%d"
-        ),
+        "year_month_day": created_at.strftime("%Y-%m-%d"),
+        "year": created_at.strftime("%Y"),
+        "month": created_at.strftime("%b"),
+        "day": created_at.strftime("%d"),
     }
 
 def create_notebook_context(notebook: Notebook, config: Config, model: str) -> dict:
